webapp/src/main/java/com/box/l10n/mojito/entity/ScheduledJob.java

```diff
@@ -0,0 +1,153 @@
+package com.box.l10n.mojito.entity;
+
+import com.box.l10n.mojito.json.ObjectMapper;
+import com.box.l10n.mojito.service.scheduledjob.ScheduledJobProperties;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.PostLoad;
+import jakarta.persistence.Table;
+import jakarta.persistence.Transient;
+import java.util.Date;
+import org.hibernate.envers.Audited;
+
+@Audited
+@Entity
+@Table(name = "scheduled_job")
+public class ScheduledJob {
+  @Id private String id;
+
+  @ManyToOne
+  @JoinColumn(name = "repository_id")
+  private Repository repository;
+
+  @ManyToOne
+  @JoinColumn(name = "job_type")
+  private ScheduledJobTypeEntity jobType;
+
+  @Column(name = "cron")
+  private String cron;
+
+  @Transient private ScheduledJobProperties properties;
```

Comment:
Out of curiosity - is the use of @Transient here to do with some desired serialization behaviour?

Reply:
Yes, the transient here indicates to the JPA that the field should not be persisted to the database. 

Once the row is pulled from the db, the deserializeProperties method is called as it has the PostLoad annotation.The properties string from the db is deserialized into this transient field. The setter of this transient field converts itself back into a JSON string which is stored in the propertiesString field that is persisted to the db.